microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -230,7 +230,8 @@ def _install(self) -> bool:
                 auth_token=self.ms_access_token,
             )
             self.env_vars["GUEST_VM_TYPE"] = self.clh_guest_vm_type
-            self.env_vars["USE_MS_GUEST_KERNEL"] = self.use_ms_guest_kernel
+            if self.use_ms_guest_kernel:
```

Comment:
we can remove the condition from `ch_tests.py` and have directly here as

```
if self.use_ms_guest_kernel == "Yes":
    self.env_vars["USE_MS_GUEST_KERNEL"] = self.use_ms_guest_kernel
```

Reply:
It would be better to put some check on value before setting it. We already have that check before executing the testcase where we set the class variable.

Reply:
In `ch_tests.py`, you are setting the variable only if `use_ms_guest_kernel` is "YES". And in `ch_tests_tool.py`, you are checking if the variable is set.  So why not just a single condition in `ch_tests_tool.py` that does "YES" check?